src/app/embed/adapters/feed.adapter.ts

```diff
@@ -92,11 +112,18 @@ export class FeedAdapter implements DataAdapter {
     return props;
   }
 
-  processEpisode(item: Element, props: AdapterProperties = {}): AdapterProperties {
+  processEpisode(item: Element, props: ReferenceableAdapterProperties = {}): ReferenceableAdapterProperties {
     props.title = this.getTagText(item, 'title');
     props.audioUrl = this.getTagTextNS(item, 'feedburner', 'origEnclosureLink')
                   || this.getTagAttribute(item, 'enclosure', 'url');
     props.artworkUrl = this.getTagAttributeNS(item, 'itunes', 'image', 'href');
+    const referenceFeedUrl = this.getTagAttributeNS(item, 'rp', 'episode-reference', 'feed-url');
```

Comment:
It's a little bit odd to have the `rp` namespace in the base feed.  I guess eventually there will be a SM tag to support similar functionality, so i like having the referenceable stuff in here.  Wondering if there's a way to get just this tag-getting into the draper adapter.